### DISCUSSION TITLE
Is there any possibility stop upgrade between 2022-02-25 and 2022-03-10？

### DISCUSSION THREAD

**Discussion by qiukeren** [filecoin-project/lotus#7991] [General Q&A]
there will be a a big festival "spring festival" in P.R.C , as important as Christmas in USA.

**Comment by Q-Action** [comment]
spring festival time is between 2022-01-31 and 2022-02-06

**Comment by SgtCoin** [comment]
Hi @qiukeren 

I do not think the team will be changing any of the network upgrade dates. In the far future, we may be able to work around certain dates, but in general, what we have now will stay on schedule.
